en/components/stepper.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Stepper - Design System Component
+_description: The Stepper Component Symbol conveys progress through numbered steps.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits, Figma, Figma to Angular, Export code from Figma, Figma HTML, Figma to HTML, Figma UI kits
+---
+
+# Stepper 
+
+Use the Stepper Component to allow the user to visualize content as a process, which shows its progress by dividing the content into successive steps. The Stepper is visually identical to the [Ignite UI for Angular Stepper Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/stepper.html)
+
+## Stepper Demo
+
+<img class="responsive-img" src="../images/stepper_demo.png" srcset="../images/stepper_demo@2x.png 2x" />
+
+## Type
+
+The Stepper component supports two types - Linear and Non Linear. In Figma you can switch between the two using the "Type" property from the properties panel. Non Linear type is used to prevent the users from proceeding with the next step, without firstly completing the previous, non-optional ones.
```

Comment:
Non Linear -> Linear type is used to prevent ...

Reply:
Fixed.